egs2/librispeech/wavlm1/local/format_pretraining.sh

```
@@ -0,0 +1,53 @@
+. ./path.sh
+
+nj=64
+fs=16k
+audio_format=flac
+dst_prefix=pretrain
+data_feats=dump/raw
+count_only=true
+
+. utils/parse_options.sh
+
+mapfile -t DSETS < <(ls -d data/${dst_prefix}_*/ 2>/dev/null \
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)